test/unit/schema/postgres.js

```diff
@@ -5,10 +5,18 @@
 var tableSql;
 
 var PG_Client = require('../../../lib/dialects/postgres');
-var client    = new PG_Client({})
+var client    = new PG_Client({});
 
 var equal  = require('assert').equal;
 
+describe("PostgreSQL Config", function(){
+  it('check version', function(){
+    var version = 'v7.2.1-0';
```

Comment:
@elhigu @RockyNiu Can't reproduce the problem locally, so not sure what's wrong with it. The assertion error itself is bogus. The assertion takes place *after* rows have been inserted, so there should be no way for the table to contain 0 rows, since the promise chain was successful... Seems to just be mariasql too?